Source/CesiumRuntime/Public/CesiumIonServer.h

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020-2023 CesiumGS, Inc. and Contributors
+
+#pragma once
+
+#include "Engine/DataAsset.h"
+#include "UObject/Object.h"
+#include "CesiumIonServer.generated.h"
+
+/**
+ * Defines a Cesium ion Server. This may be the public (SaaS) Cesium ion server
+ * at ion.cesium.com, or it may be a self-hosted instance.
+ */
+UCLASS()
+class CESIUMRUNTIME_API UCesiumIonServer : public UDataAsset {
+  GENERATED_BODY()
+
+public:
+  /**
+   * Gets the default Cesium ion Server (ion.cesium.com).
+   *
+   * It is expected to be found at
+   * `/Game/CesiumSettings/CesiumIonServers/CesiumIonSaaS`. In the Editor, it
+   * will be created if it does not already exist, so this method always returns
+   * a valid instance. At runtime, this method returns nullptr if the object
+   * does not exist.
+   */
+  static UCesiumIonServer* GetDefault();
```

Comment:
Isn't that a little redundant when they'll be accessed with:

```
UCesiumIonServer* pServer = UCesiumIonServer::GetDefault();
```

I'm not strongly opposed, but it seems unnecessary. Can you point me to the other objects where we're doing similar things?

Reply:
Sure. In `CesiumGeoreference.h`:

```
  UFUNCTION(
      BlueprintCallable,
      Category = "Cesium",
      meta = (WorldContext = "WorldContextObject"))
  static ACesiumGeoreference*
  GetDefaultGeoreference(const UObject* WorldContextObject);
```

In `CesiumCreditSystem.h`:
```
  static ACesiumCreditSystem*
  GetDefaultCreditSystem(const UObject* WorldContextObject);
```

You could argue that `CesiumGeoreference` does this for the sake of clarity when using Blueprints. But I saw that `CesiumCreditSystem` was doing this as well, so I thought this was a stylistic choice. Personally, I like more explicit function names because they help with the initial read of the code, but it's not a major problem. 

If you're willing to accept the change, though, I'm willing to do the name replacements myself!

Reply:
Ok, I'm convinced, I'll change it. Thanks!